prometheus/src/main/scala/filodb/prometheus/parse/Parser.scala

```diff
@@ -53,6 +53,10 @@ object Parser extends StrictLogging {
       case Antlr => AntlrParser.parseQuery(query)
       case Legacy => LegacyParser.parseQueryWithPrecedence(query)
       case Shadow => {
+        // TODO(a_theimer): should this be changed to something like:
+        //   if (both parse) then { assert match }
+        //   else if (none parse) then { throw }
+        //   else { just log the failure and hope for the best }
```

Comment:
I think my question can be reworded as: should I also update the `LegacyParser`?

Reply:
double-check that ANTLR runs